bodhi/server/mail.py

```diff
@@ -281,10 +281,40 @@
 
 --------------------------------------------------------------------------------
 %(notes)s%(changelog)s%(references)s
+This update can be installed with the "dnf" update programs.  Use
+su -c 'dnf%(yum_repository)s update %(name)s' at the command line.
+For more information, refer to "Package Management",
+available at https://docs.fedoraproject.org
+
+All packages are signed with the Fedora EPEL GPG key. More details on the
+GPG keys used by the Fedora Project can be found at
+https://fedoraproject.org/keys
+--------------------------------------------------------------------------------
+"""
+
+# message template for legacy systems, still pointing users to use "yum"
+fedora_epel_legacy_errata_template = u"""\
```

Comment:
Actually, on second thought I see that your if statement below works with EL 8 already. I'm convinced that what I just wrote is no good, so please do keep these ☺